lib/gettext_i18n_rails.rb

```diff
@@ -2,6 +2,15 @@
 require 'gettext_i18n_rails/gettext_hooks'
 
 module GettextI18nRails
+  IGNORE_TABLES = [/^sitemap_/, /_versions$/, 'schema_migrations', 'sessions', 'delayed_jobs']
+
+  def self.ignore_tables
+    IGNORE_TABLES
+  end
+
+  def self.add_ignore_table(str)
```

Comment:
no need to care about the defaults when using `GettextI18nRails::IGNORE_TABLES << 'my_table'` :)